roles/generate-docs/tasks/create_readme.yml

```diff
@@ -0,0 +1,12 @@
+---
+# Attempted to generate docs with the uri module like:
+# https://github.com/redhat-cop/infra-ansible/blob/master/roles/ansible/tower/manage-inventories/tasks/process-group.yml
+# But recieved error "unautorized". Is this method specifically for ansible tower?
+
+# Also, attempted using the pandoc cli tool like:
```

Comment:
No need for `pandoc` here - i.e.: what you have below is very much the correct direction - so let's remove these comments.